src/code/z_player_call.c

```diff
@@ -1,6 +1,7 @@
 #include "global.h"
 
-#define FLAGS (ACTOR_FLAG_0 | ACTOR_FLAG_2 | ACTOR_FLAG_4 | ACTOR_FLAG_5 | ACTOR_FLAG_25 | ACTOR_FLAG_26)
+#define FLAGS \
+    (ACTOR_FLAG_ATTENTION_ENABLED | ACTOR_FLAG_HOSTILE | ACTOR_FLAG_4 | ACTOR_FLAG_5 | ACTOR_FLAG_25 | ACTOR_FLAG_26)
 
```

Comment:
Player hates himself 😢 
Very loosely I think this could be evidence that dark link was once a "real" player, and didnt have the torch2 wrapper layer. But obviously its not very concrete evidence.

Anyway it doesn't matter either way. The Attention system doesn't search the Player category, and he cant lock onto himself

Reply:
oh hey mm fixed it 
![image](https://github.com/user-attachments/assets/f31df732-1f58-4121-9156-f755512f04c2)